heka/sandbox/filters/firefox_plugin_hangs.lua

```diff
@@ -28,22 +28,26 @@ local BUILD_IDS  = 6 -- number of builds to display on the graph (newer builds r
 local DAYS       = 180
 local SEC_IN_DAY = 60 * 60 * 24
 
-local function set_header(hph, cph, col, id)
-    hph:set_header(col, id, "hangs/hour", "none")
-    cph:set_header(col, id, "crash/hour", "none")
+local function set_headers(channel, col, id)
+    channel.hph:set_header(col, id, "hangs/hour", "none")
+    channel.cph:set_header(col, id, "crash/hour", "none")
+    channel.hours:set_header(col, id)
+    channel.hangs:set_header(col, id)
+    channel.crashes:set_header(col, id)
 end
 
 local channel_cnt = fx.get_channel_count()
 for i=1, channel_cnt do
-    local hph = circular_buffer.new(DAYS, BUILD_IDS, SEC_IN_DAY)   -- hangs per hour
-    local cph = circular_buffer.new(DAYS, BUILD_IDS, SEC_IN_DAY)   -- crashes per hour
-    local hours = circular_buffer.new(DAYS, BUILD_IDS, SEC_IN_DAY) -- total hours of use
-    local hangs = circular_buffer.new(DAYS, BUILD_IDS, SEC_IN_DAY) -- total number of hangs
-    local crashes = circular_buffer.new(DAYS, BUILD_IDS, SEC_IN_DAY) -- total number of crashes
-    channels[fx.get_channel_name(i-1)] = {hph = hph, cph = cph, hours = hours, hangs = hangs, crashes = crashes, ids = {}}
+    local hph = circular_buffer.new(DAYS, BUILD_IDS, SEC_IN_DAY) -- hangs per hour
```

Comment:
Might be good to add a comment here that we do not enable deltas for hourly rates (where they cannot be aggregated), but *do* enable deltas for the totals.